metadata.yaml

```diff
@@ -1,350 +0,0 @@
-missions:
```

Comment:
I thought this file would stay? Or is it gonna be converted to some other format as well?

Reply:
Yes, it stays, but as a JSON file now. That way everything is easier parseable by most types of clients.

Reply:
Gotcha, thanks!